package.json

```diff
@@ -21,14 +20,16 @@
   },
   "private": true,
   "dependencies": {
-    "@educandu/educandu": "0.85.3",
-    "@educandu/node-jsx-loader": "~3.0.0",
+    "@educandu/educandu": "~0.86.0",
+    "@educandu/eslint-config": "~2.0.0",
+    "@educandu/node-jsx-loader": "~3.0.1",
+    "del": "~7.0.0",
     "parseboolean": "~1.0.0"
   },
   "devDependencies": {
-    "@educandu/dev-tools": "~4.0.0",
-    "@educandu/rooms-auth-lambda": "~2.0.1",
-    "eslint": "~8.29.0",
+    "@educandu/dev-tools": "~6.0.2",
+    "@educandu/rooms-auth-lambda": "~2.0.3",
+    "eslint": "~8.30.0",
     "eslint-plugin-import": "~2.26.0",
```

Comment:
I had to remove `eslint-plugin-import` completely, as it was buggy like hell with false positives. So this dependency can also go.

Reply:
Actually it's no longer used with the eslint config change. Thanks, removed!